systemvm/debian/opt/cloud/bin/cs/CsRedundant.py

```diff
@@ -280,7 +280,7 @@ def set_backup(self):
             dev = interface.get_device()
 
         cmd = "%s -C %s" % (self.CONNTRACKD_BIN, self.CONNTRACKD_CONF)
```

Comment:
@weizhouapache if `cmd` is unused can we remove it?

Reply:
@rhtyd sure...